lesson01/assignment/README.md

```diff
@@ -2,6 +2,6 @@
 
 ## Assignment 1
 * Create at least four forms with input boxes and submit buttons.
-* Use ES6+ array and object methods to manipulate the data input, and ouput the result with an explanation.
+* Use ES6+ array and object methods to manipulate the data input, and output the result with an explanation.
```

Comment:
Nice catch! You should do a pull request back to the UW repo with just this change 😄